### PR TITLE
Aimp.aimp v5.03.2394

### PR DESCRIPTION
**Pull request by hamnarauf**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/65988)

### CODE REVIEW

File: manifests/a/AIMP/AIMP/v5.03.2394, 30.06.2022/AIMP.AIMP.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AIMP.AIMP
+PackageVersion: v5.03.2394, 30.06.2022
```
Comment (ItzLevvie) [inline manifests/a/AIMP/AIMP/v5.03.2394, 30.06.2022/AIMP.AIMP.installer.yaml:5]:
```suggestion
PackageVersion: 5.03.2394
```
Looks like they changed their versioning scheme so this must match Control Panel.

File: manifests/a/AIMP/AIMP/v5.03.2394, 30.06.2022/AIMP.AIMP.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AIMP.AIMP
+PackageVersion: v5.03.2394, 30.06.2022
```
Comment (ItzLevvie) [inline manifests/a/AIMP/AIMP/v5.03.2394, 30.06.2022/AIMP.AIMP.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 5.03.2394
```
Looks like they changed their versioning scheme so this must match Control Panel.

File: manifests/a/AIMP/AIMP/v5.03.2394, 30.06.2022/AIMP.AIMP.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: AIMP.AIMP
+PackageVersion: v5.03.2394, 30.06.2022
```
Comment (ItzLevvie) [inline manifests/a/AIMP/AIMP/v5.03.2394, 30.06.2022/AIMP.AIMP.yaml:5]:
```suggestion
PackageVersion: 5.03.2394
```
Looks like they changed their versioning scheme so this must match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/a/AIMP/AIMP/5.03.2394/AIMP.AIMP.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+PackageVersion: 5.03.2394
+PackageLocale: en-US
+Publisher: AIMP DevTeam
+PublisherUrl: https://www.aimp.ru
+PublisherSupportUrl: https://www.aimp.ru/forum/index.php
+# PrivacyUrl: 
+Author: Artem Izmaylov
+PackageName: AIMP
+PackageUrl: https://www.aimp.ru
+License: EULA
```
Comment (Trenly) [inline manifests/a/AIMP/AIMP/5.03.2394/AIMP.AIMP.locale.en-US.yaml:14]:
```suggestion
License: Proprietary
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Pull request merged**